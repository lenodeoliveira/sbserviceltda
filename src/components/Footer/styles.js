import styled, { css } from 'styled-components'
import { Container as SectionContainer } from '../SectionContainer'

export const Container = styled.footer`
  ${({ theme }) => css`
    text-align: center;
    border-top: 0.1rem solid ${theme.colors.mediumGray};
    p {
      font-size: 1.8rem;
    }
    & ${SectionContainer} {
      padding-top: 0;
      padding-bottom: 0;
    }
  `}
`
