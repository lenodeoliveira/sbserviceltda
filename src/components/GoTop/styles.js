import styled, { css } from 'styled-components'

export const Container = styled.a`
  ${({ theme }) => css`
    position: fixed;
    bottom: 2rem;
    right: 2rem;
    z-index: 6;
    color: ${theme.colors.white};
    background: ${theme.colors.primaryColor};
    display: flex;
    align-items: center;
    justify-content: center;
    width: 4rem;
    height: 4rem;
    opacity: 0.8;
  `}
`
