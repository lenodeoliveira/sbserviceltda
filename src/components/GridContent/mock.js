export default {
  title: 'Grid content',
  text: `Lorem ipsum dolor sit amet consectetur adipisicing elit. Distinctio magnam culpa eveniet doloribus harum? Ipsam, a necessitatibus? Sequi sunt accusantium quod, animi iure a, aliquid dolor ea vel magni dolore?`,
  background: false,
}
