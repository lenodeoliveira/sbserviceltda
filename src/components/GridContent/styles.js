import styled, { css } from 'styled-components'

export const Container = styled.div`
  ${() => css`
    text-align: center;
    max-width: 58rem;
    margin: 0 auto;
  `}
`
export const TextContainer = styled.div`
  ${({ theme }) => css`
    margin: ${theme.spacings.xhuge} 0;
  `}
`
