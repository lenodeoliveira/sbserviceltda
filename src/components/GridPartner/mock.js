export default {
  background: false,
  title: 'My grid',
  description: 'Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde',
  grid: [
    {
      altText: 'Teste 1',
      srcImage: 'assets/images/partner.jpeg',
    },
    {
      altText: 'Teste 2',
      srcImage: 'assets/images/partner.jpeg',
    },
    {
      altText: 'Teste 3',
      srcImage: 'assets/images/partner.jpeg',
    },
    {
      altText: 'Teste 4',
      srcImage: 'assets/images/partner.jpeg',
    },
    {
      altText: 'Teste 5',
      srcImage: 'assets/images/partner.jpeg',
    },
    {
      altText: 'Teste 6',
      srcImage: 'assets/images/partner.jpeg',
    },
  ],
}
