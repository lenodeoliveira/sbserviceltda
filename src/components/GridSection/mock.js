export default {
  background: false,
  title: 'My grid',
  description: 'Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde',
  grid: [
    {
      title: 'Teste 1',
      description: 'Lorem ipsum dolor sit,.',
    },
    {
      title: 'Teste 2',
      description: 'Lorem ipsum dolor sit,',
    },
    {
      title: 'Teste 3',
      description: 'Lorem ipsum dolor sit, ',
    },
    {
      title: 'Teste 3',
      description: 'Lorem ipsum dolor sit, ',
    },
  ],
}
