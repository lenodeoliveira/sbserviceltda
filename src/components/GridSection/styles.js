import styled, { css } from 'styled-components'
import { Title as HeadingContainer } from '../Heading/styles'
import { Container as TextComponent } from '../TextComponent/styles'

export const Container = styled.div`
  ${({ theme }) => css`
    ${TextComponent} {
      margin-bottom: ${theme.spacings.xhuge};
    }
  `}
`
export const Grid = styled.div`
  ${({ theme }) => css`
    display: grid;
    grid-template-columns: repeat(auto-fill, minmax(280px, 1fr));
    gap: ${theme.spacings.medium};
  `}
`
export const GridElement = styled.div`
  ${() => css`
    ${HeadingContainer} {
    }
  `}
`
