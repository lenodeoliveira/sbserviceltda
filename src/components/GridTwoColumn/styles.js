import styled, { css } from 'styled-components'
import { Title } from '../Heading/styles'

export const Container = styled.div`
  ${({ theme }) => css`
    display: grid;
    grid-template-columns: 1.8fr 1fr;
    margin-left: 3rem;
    align-items: center;
    gap: ${theme.spacings.large};

    @media ${theme.media.lteMedium} {
      grid-template-columns: 1fr;
      text-align: center;
      margin-left: 0;
    }
    ${Title} {
      margin-bottom: ${theme.spacings.xlarge};
    }
  `}
`
export const TextContainer = styled.div`
  ${({ theme }) => css`
    @media ${theme.media.lteMedium} {
      margin-bottom: ${theme.spacings.large};
    }
  `}
`
export const ImageContainer = styled.div`
  ${() => css`
    width: 100%;
  `}
`
export const Image = styled.img`
  ${() => css`
    width: 100%;
    border-radius: 2.4%;
  `}
`
