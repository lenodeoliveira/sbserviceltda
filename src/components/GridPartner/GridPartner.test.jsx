import { GridPartner } from '.'
import { renderTheme } from '../../styles/render-theme'

import mock from './mock'

describe('<GridPartner />', () => {
  it('Should render GridPartner', () => {
    const { container } = renderTheme(<GridPartner {...mock} />)
    expect(container).toMatchSnapshot()
  })
})
