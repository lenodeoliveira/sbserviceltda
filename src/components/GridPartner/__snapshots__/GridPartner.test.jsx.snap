// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridPartner /> Should render GridPartner 1`] = `
.c5 {
  font-size: 2.4rem;
}

.c2 > .c4 {
  margin-bottom: 6.4rem;
}

.c6 {
  display: grid;
  grid-template-columns: repeat(auto-fill,minmax(280px,1fr));
  gap: 2.4rem;
}

.c7 {
  overflow: hidden;
}

.c8 {
  width: 100%;
}

.c0 {
  background: #FFFFFF;
  color: #2F5233;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 {
  max-width: 120rem;
  margin: 0 auto;
  padding: 3.2rem;
  width: 100%;
}

.c3 {
  color: #2F5233;
  font-size: 6.4rem;
  text-transform: uppercase;
  font-family: 'Montserrat',sans-serif;
}

@media (max-width:768px) {
  .c6 {
    grid-template-columns: 1fr;
  }
}

@media (max-width:768px) {
  .c3 {
    font-size: (max-width:768px);
  }
}

<div>
  <div
    class="c0"
    id=""
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <h2
          class="c3"
        >
          My grid
        </h2>
        <p
          class="c4 c5"
        >
          Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde
        </p>
        <div
          class="c6"
        >
          <div
            class="c7"
          >
            <img
              alt="Teste 1"
              class="c8"
              src="assets/images/partner.jpeg"
            />
          </div>
          <div
            class="c7"
          >
            <img
              alt="Teste 2"
              class="c8"
              src="assets/images/partner.jpeg"
            />
          </div>
          <div
            class="c7"
          >
            <img
              alt="Teste 3"
              class="c8"
              src="assets/images/partner.jpeg"
            />
          </div>
          <div
            class="c7"
          >
            <img
              alt="Teste 4"
              class="c8"
              src="assets/images/partner.jpeg"
            />
          </div>
          <div
            class="c7"
          >
            <img
              alt="Teste 5"
              class="c8"
              src="assets/images/partner.jpeg"
            />
          </div>
          <div
            class="c7"
          >
            <img
              alt="Teste 6"
              class="c8"
              src="assets/images/partner.jpeg"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
